cmd/openshift-install/create.go

```diff
@@ -104,6 +104,7 @@ var (
 				}
 				consoleURL, err := waitForConsole(ctx, config, rootOpts.dir)
 				if err != nil {
+					logrus.Errorf("Error: %v", err)
```

Comment:
I'd rather log this further up the stack.  If we're returning it to the caller, and the caller decides to ignore it, it should log a warning.  If the caller decides to die (and on up the stack) we will eventually end up in a `logrus.Fatal` or similar when the program exits.